Set up C4 page ready for fellows, moved application page to new C5 page

alexandertdeng opened the pull request:
Made the fellow page URLs and picture URLs dynamically generated on the layout pages, instead of static links from the YML file which are time-consuming to generate per fellow in future cohorts. Several photos changed to JPG from PNG to point to correct paths. Removed or replaced a couple of dead links in fellow profiles

alexandertdeng committed:
Set up C4 page ready for fellows, moved application page to new C5 page

Made the fellow page URLs and picture URLs dynamically generated on the layout pages, instead of static links from the YML file which are time-consuming to generate per fellow in future cohorts.
Several photos changed to JPG from PNG to point to correct paths.
Removed or replaced a couple of dead links in fellow profiles

alexandertdeng committed:
Populated cohort 4 yml file with data

alexandertdeng committed:
Fellow images rationalised

Removed per fellow image URL- now dynamically generated from _layout for fellows
Slugified image name per fellow implemented

alexandertdeng committed:
C4 fellow pages generated, awaiting cards to be added

alexandertdeng committed:
C4 headshot photos added

alexandertdeng committed:
Update katherine-wykes.jpg

alexandertdeng committed:
Fixed Ala's AlAqeel named to display as preferred by fellow

dangerdika reviewed (approved):
Approving this but there are a number of conflicts to be addressed when it comes to merging. The new cohort has pictures but no cards so the pages you click on load with a broken image link instead of blank - should there be a different placeholder?

alexandertdeng committed:
Merge branch 'gh-pages' into 586-fcai-c4-website-cards-upload-and-fellow-pages

alexandertdeng committed:
Converting fellowship cohort yml and _includes files to new socials format

Rationalised some naming conventions for url generation in other parts of the site

alexandertdeng committed:
Fixed typo on team _includes page

alexandertdeng committed:
Added quote cards for C4 fellows

Also minor path corrections for international fellows
